examples/Planning/ForecastReach.php

```diff
@@ -0,0 +1,384 @@
+                $point->getForecast()->getOnTargetImpressionsUnwrapped(),
+                $point->getForecast()->getTotalReachUnwrapped(),
+                $point->getForecast()->getTotalImpressionsUnwrapped(),
+                $products,
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Pulls a forecast for product mix created manually.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param string $locationId the location ID to plan for. You can get a valid location ID from
+     *     https://developers.google.com/adwords/api/docs/appendix/geotargeting or
+     *     by calling ListPlannableLocations on the ReachPlanService.
+     * @param string $currencyCode three-character ISO 4217 currency code
+     * @param float $budget_micros budget to allocate to the plan
+     */
+    private static function forecastManualMix(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $locationId,
+        string $currencyCode,
+        float $budget_micros
+    ) {
+        // Set up a ratio to split the budget between two products.
+        $trueviewAllocation = floatval(0.15);
+        $bumperAllocation = floatval(1 - $trueviewAllocation);
+
+        // See listPlannableProducts on ReachPlanService to retrieve a list
+        // of valid PlannableProductCode's for a given location:
+        // https://developers.google.com/google-ads/api/reference/rpc/Google.Ads.GoogleAds.V3.services#reachplanservice
+        $productMix = [
+            new PlannedProduct([
+                'plannable_product_code' => new StringValue(['value' => 'TRUEVIEW_IN_STREAM']),
+                'budget_micros' => new Int64Value([
+                    'value' => $budget_micros * $trueviewAllocation
+                ])
+            ]),
+            new PlannedProduct([
+                'plannable_product_code' => new StringValue(['value' => 'BUMPER']),
+                'budget_micros' => new Int64Value([
+                    'value' => $budget_micros * $bumperAllocation
+                ])
+            ])
+        ];
+
+        self::requestReachCurve(
+            $googleAdsClient,
+            $customerId,
+            $productMix,
+            $locationId,
+            $currencyCode
+        );
+    }
+
+    /**
+     * Pulls a forecast for a product mix based on your set of preferences.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param string $locationId the location ID to plan for. You can get a valid location ID from
+     *     https://developers.google.com/adwords/api/docs/appendix/geotargeting or
+     *     by calling ListPlannableLocations on the ReachPlanService.
+     * @param string $currencyCode three-character ISO 4217 currency code
+     * @param float $budget_micros budget to allocate to the plan
```

Comment:
Ditto. budgetMicros
Actually, if you pass const directly, this will be gone though.